### PR TITLE
feat-server-aws-publish

### PR DESCRIPTION
**Pull request by ryanakiyama**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,6 +40,7 @@
     "eslint-plugin-import": "^2.8.0",
     "eslint-plugin-jsx-a11y": "^6.0.2",
     "eslint-plugin-react": "^7.4.0",
+    "webpack": "^2.2.1"
```
Comment (christine-w) [inline package.json]: Please take a look.

File: schema.sql [hunk 2]
```diff
@@ -65,16 +65,21 @@ CREATE TABLE availability (
...
-INSERT INTO neighborhoods (neighborhood_name, destinations_id) VALUES ("Golden Gate Park", 1);
-INSERT INTO neighborhoods (neighborhood_name, destinations_id) VALUES ("Haight Ashbury", 1);
-INSERT INTO neighborhoods (neighborhood_name, destinations_id) VALUES ("Inner Richmond", 1);
-INSERT INTO neighborhoods (neighborhood_name, destinations_id) VALUES ("Inner Sunset", 1);
-INSERT INTO neighborhoods (neighborhood_name, destinations_id) VALUES ("Outer Richmond", 1);
-INSERT INTO neighborhoods (neighborhood_name, destinations_id) VALUES ("Outer Sunset", 1);
-INSERT INTO neighborhoods (neighborhood_name, destinations_id) VALUES ("Presidio", 1);
-INSERT INTO neighborhoods (neighborhood_name, destinations_id) VALUES ("Financial District", 1);
+INSERT INTO destinations (destination_name) VALUES ('San Francisco');
+INSERT INTO neighborhoods (neighborhood_name, destinations_id) VALUES ('Glen Park', 1);
```
Comment (christine-w) [inline schema.sql:69]: Suggestion: There is an opportunity here to reduce the repetition with a loop.

File: server/aws.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
+const AWS = require('aws-sdk');
+
+AWS.config.update({
+  accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+});
+
+const sqs = new AWS.SQS({ region: 'us-west-1' });
+
+const inventoryMessage = { topic: 'Inventory', payload: 'Test from Inventory' };
```
Comment (christine-w) [inline server/aws.js]: Suggestion: Repetition here can be reduced by having a common message object and differentiating by payload.

File: server/aws.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+const searchMessage = { topic: 'Inventory', payload: 'Test from Inventory' };
+const recommendationsMessage = { topic: 'Inventory', payload: 'Test from Inventory' };
+
+const sqsParamsInventory = {
```
Comment (christine-w) [inline server/aws.js]: Suggestion: Repetition here can be reduced with a helper function that constructs this object using message and queue url.

File: server/aws.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  MessageBody: JSON.stringify(searchMessage),
+  QueueUrl: process.env.SEARCH_QUEUE_URL,
+};
+
+const sqsParamsRecommendations = {
+  MessageBody: JSON.stringify(recommendationsMessage),
+  QueueUrl: process.env.RECOMMENDATIONS_QUEUE_URL,
+};
+
+sqs.sendMessage(sqsParamsInventory, (err, data) => {
```
Comment (christine-w) [inline server/aws.js]: Suggestion: Same method run with different arguments -> opportunity for DRY with a loop.

**Review by christine-w — APPROVED** [review]

File: schema.sql (hunk 2 above)
Comment (ryanakiyama) [reply schema.sql:69]: Will leave this for now until I find a way to create loops in .sql files

File: server/aws.js (hunk 3 above)
Comment (ryanakiyama) [reply server/aws.js]: Will refactor when functionality to consume messages from aws sqs is implemented, just pushed up code to show functionality to publish messages to aws sqs

**Pull request merged**